config/index.js

```diff
@@ -285,5 +288,46 @@ module.exports = {
     'wrap-regex': 'error',
     'yield-star-spacing': ['error', 'both'],
     'yoda': ['error', 'never'],
+
+    'import/default': 'error',
+    'import/dynamic-import-chunkname': 'off',
+    'import/export': 'error',
+    'import/exports-last': 'off',
+    'import/extensions': ['error', 'never', { 'json': 'always' }],
+    'import/first': 'error',
+    'import/group-exports': 'off',
+    'import/max-dependencies': 'off',
+    'import/named': 'error',
+    'import/namespace': 'error',
+    'import/newline-after-import': ['error', { 'count': 1 }],
```

Comment:
Nit: A count of 1 is the default option
```suggestion
    'import/newline-after-import': 'error',
```